lib/stack.dart

```diff
@@ -30,4 +30,12 @@ class Stack<T> {
   int size() {
     return _list.length;
   }
+
+  // print stack
+  void printStack() {
+    while (!_list.isEmpty) {
+      print(_list.last);
+      _list.removeLast();
```

Comment:
@AunSyedShah Here, print will empty the stack

I think you should loop through it instead of that

Reply:
I checked it, this implementation is working with cpp and dart. @ammaratef45. Do let me know with your kind suggestions further.